lib/order.rb

```diff
@@ -0,0 +1,26 @@
+require 'forwardable'
+
+class Order
+
+  extend Forwardable
+
+  def_delegator :@menu, :list_items, :menu_items
+
+  attr_reader :customer, :menu, :order_details
+
+  def initialize customer, menu, order_details
+    @customer = customer
+    @menu = menu
+    @order_details = order_details
+    @items = []
+    order_details.each do |i|
+      @items << i unless menu_items.keys.include? i
+    end
+    fail "Sorry, we don't serve: #{@items.join(', ')}. Order aborted." unless @items.empty?
```

Comment:
Metrics/LineLength: Line is too long. [91/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)